e2e/main.go

```diff
@@ -3,12 +3,12 @@ package main
 import (
 	"flag"
 	"fmt"
-	"os"
-	"path/filepath"
-
 	"github.com/MontFerret/ferret/e2e/runner"
 	"github.com/MontFerret/ferret/e2e/server"
 	"github.com/rs/zerolog"
+	"os"
```

Comment:
Here you can use ```goimports``` util.

Reply:
@3timeslazy Could you give me more details on that?

Reply:
https://github.com/bradfitz/goimports